content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: With the Telemetry API [Experimental]
+description: How to configure tracing options with the Telemetry API (experimental).
+weight: 10
+keywords: [telemetry,tracing]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Istio provides a Telemetry API that enables flexible configuration of tracing behavior. The experimental
+Telemetry API offers controls over tracing options such as sampling rates and custom tags for individual
+spans, as well as backend provider selection. The options are configurable down to the the individual
+workload level.
+
+## Before you begin
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
+1.  Follow the tracing installation guide located under [Integrations](/docs/ops/integrations/) based on your preferred tracing backend
+    to install the appropriate tracing provider.
+
+## Telemetry API: Tracing Overview
+
+The Telemetry API offers tracing behavior configuration control over the following at the mesh, namespace, and workload levels:
+
+- **provider selection** - allows selection of backend providers (providers are configured in `MeshConfig`) for reporting.
+
+- **sampling percentage** - allows control the rate of trace sampling applied to received requests *for which no prior sampling decision has been made*.
```

Comment:
If no sampling decision has been made, it applies. If the prior hop chose not to sample, then the request will be marked as not sampled, and that will be passed along to all subsequent hops. This will not override that decision.